tests/test_integrated_channels/test_moodle/test_exporters/test_content_metadata.py

```diff
@@ -104,7 +104,7 @@ def test_transform_description_includes_url(self, content_metadata_item, expecte
     @responses.activate
     def test_transform_start(self):
         """
-        `MoodleContentMetadataExporter``'s ``transform_start` returns int timestamp.
+        `MoodleContentMetadataExporter``'s ``transform_start`` returns int timestamp.
```

Comment:
good find haha

Reply:
I honestly don't know what these backticks are here for anyways. I just wanted the format to be consistent.